### PR TITLE
Add some fields to the scraper's manifest

### PR DESCRIPTION
**Pull request by SamDepardieu**
# Description
Add's start-time, end-time and organisation fields to the manifest

Also ignore those fields in the pdf parser task.

## Type of change
- [x] :sparkles: New feature

# How Has This Been Tested?
Local tests
`make test`
`make docker-test`

### CODE REVIEW

File: policytool/pdf_parser/main.py [hunk 1]
```diff
@@ -231,21 +238,25 @@ def parse_all_pdf(input_url, output_url, organisation, context,
...
-                item = parse_pdf(
-                    tf.name,
-                    words,
-                    titles,
-                    context,
-                    item,
-                )
-            parsed_items.append(item)
+        if directory in MANIFEST_IGNORE_FIELDS:
+            continue
```
Comment (hblanks) [inline policytool/pdf_parser/main.py]: Could we nest the manifest instead, please? I.e.:

```python
manifest = {
    'metadata': {...}',
    'content': {
        'aa': [],
    }
}
```

or such?

**Comment by SamDepardieu** [comment]
@hblanks Updated and tested - clean and working 👍

File: policytool/scraper/wsf_scraping/file_system.py [hunk 2]
```diff
@@ -118,15 +120,27 @@ def get_manifest(self):
 
     def update_manifest(self, data_file):
         current_manifest = self.get_manifest()
+
+        metadata = {}
+        metadata['organisation'] = self.organisation
+        metadata['start-time'] = \
+            self.start_time.strftime("%Y-%m-%d, %H:%M:%S")
+        metadata['stop-time'] = \
+            datetime.datetime.now().strftime("%Y-%m-%d, %H:%M:%S")
```
Comment (hblanks) [inline policytool/scraper/wsf_scraping/file_system.py]: Can you please update this to ISO8601? Easiest way would be `self.start_time.isoformat()`, `datetime.datetime.now().isoformat()`?

**Comment by hblanks** [comment]
One more note - do we need to update anything downstream as well, or does the filesystem take care of this? (Seems likely we might need to..)

**Comment by SamDepardieu** [comment]
Changed to iso as requested. What do you mean by downstream? The only consumer of this manifest at the moment is the pdf parser task, and this is already handled in this PR

**Pull request merged**